pandas-stubs/core/groupby/groupby.pyi

```diff
@@ -357,7 +356,6 @@ class GroupByPlot(PlotAccessor, Generic[_GroupByT]):
 
 class BaseGroupBy(SelectionMixin[NDFrameT], GroupByIndexingMixin):
     axis: AxisInt
-    grouper: ops.BaseGrouper
     keys: _KeysArgType | None
     level: IndexLabel | None
     group_keys: bool
```

Comment:
The proverbial Pandora's box.  Now we can see that we should delete these 4 declarations as well since they are not documented.

Reply:
Indeed that seems like it opened to more cleaning, I did a few more deletions in that file since they were all not documented.